tests/readmes/test_fre_flow.py

```diff
@@ -0,0 +1,110 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import os
+
+from ocean_lib.example_config import ExampleConfig
+from ocean_lib.models.erc20_token import ERC20Token
+from ocean_lib.models.models_structures import ErcCreateData
+from ocean_lib.ocean.mint_fake_ocean import mint_fake_OCEAN
+from ocean_lib.ocean.ocean import Ocean
+from ocean_lib.web3_internal.constants import ZERO_ADDRESS
+from ocean_lib.web3_internal.currency import to_wei
+from ocean_lib.web3_internal.wallet import Wallet
+
+
+def test_fre_flow():
+    """This test mirrors the Fixed Rate Exchange README.
+    As such, it does not use the typical pytest fixtures.
+    """
+
+    config = ExampleConfig.get_config()
+    ocean = Ocean(config)
+
+    # Create Alice's wallet
+    alice_private_key = os.getenv("TEST_PRIVATE_KEY1")
+    alice_wallet = Wallet(
+        ocean.web3,
+        alice_private_key,
+        config.block_confirmations,
+        config.transaction_timeout,
+    )
+    assert alice_wallet.address
+
+    # Mint OCEAN
+    mint_fake_OCEAN(config)
+    assert alice_wallet.web3.eth.get_balance(alice_wallet.address) > 0, "need ETH"
+
+    # Publish an NFT token
+    nft_token = ocean.create_nft_token(
+        "NFTToken1", "NFT1", alice_wallet, "https://oceanprotocol.com/nft/"
+    )
+    token_address = nft_token.address
+    assert token_address
+
+    # Prepare data for ERC20 token
+    erc20_data = ErcCreateData(
+        template_index=1,
+        strings=["Datatoken 1", "DT1"],
+        addresses=[
+            alice_wallet.address,
+            alice_wallet.address,
+            ZERO_ADDRESS,
+            ocean.OCEAN_address,
+        ],
+        uints=[to_wei(200), 0],
+        bytess=[b""],
+    )
+
+    erc20_token = nft_token.create_datatoken(erc20_data, alice_wallet)
+    print(f"token_address = '{erc20_token.address}'")
+
+    # Mint the datatokens
+    erc20_token.mint(alice_wallet.address, to_wei(100), alice_wallet)
+
+    # Bob buys at fixed rate data tokens
+    bob_private_key = os.getenv("TEST_PRIVATE_KEY2")
+    bob_wallet = Wallet(
+        ocean.web3,
+        bob_private_key,
+        config.block_confirmations,
+        config.transaction_timeout,
+    )
+    print(f"bob_wallet.address = '{bob_wallet.address}'")
+
+    # Verify that Bob has ganache ETH
+    assert ocean.web3.eth.get_balance(bob_wallet.address) > 0, "need ganache ETH"
+
+    OCEAN_token = ERC20Token(ocean.web3, ocean.OCEAN_address)
```

Comment:
please remove the ERC20Token import and use the function in the Ocean class to retrieve the token.